s3_management/manage.py

```diff
@@ -104,6 +104,7 @@
     "zipp",
     # ----
     "Pillow",
+    "pillow",
```

Comment:
Should we just replace one with another?
```suggestion
    "pillow",
```